presto-hive/src/main/java/com/facebook/presto/hive/HiveKerberosConfig.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.NotNull;
+
+public class HiveKerberosConfig
```

Comment:
Having these in the same file seems wrong, since it disallows configuring HDFS and Metastore independently. I think either of these configs will break:
```
hive.metastore.authentication.type=NONE
hive.hdfs.authentication.type=KERBEROS
```
or
```
hive.metastore.authentication.type=KERBEROS
hive.hdfs.authentication.type=NONE
```
These should probably be two separate configs, `HdfsKerberosConfig` and `MetastoreKerberosConfig`.